packages/driver/cypress/e2e/commands/actions/trigger.cy.js

```diff
@@ -608,7 +598,19 @@ describe('src/cy/commands/actions/trigger', () => {
         cy.get('button:first').trigger('tap', { animationDistanceThreshold: 1000 }).then(() => {
           const { args } = cy.ensureElementIsNotAnimating.firstCall
 
-          expect(args[1]).to.deep.eq([fromElWindow, fromElWindow])
+          args[1].forEach((fromElWin) => {
+            // in some firefox versions, leftCenter may drift slightly by a pixel. The reason for this is unclear.
```

Comment:
not entirely sure why this is happening, but some versions of firefox are experiencing as much as a pixel drift in some of these properties, likely due to the changes we made to the `dom` files within the `driver`

Reply:
I think I've seen flaky tests around this in general

Reply:
except this fails every single time in firefox 93 it appears. I wonder if its related and we just managed to make it worse?

Reply:
I've seen failures like this before. Firefox reports positions/scroll offsets with something like 13 digits of precision. We do a lot rounding in our [coordinate](https://github.com/cypress-io/cypress/blob/3e6d6bfe155234f782a890fd904b361c84d5c34e/packages/driver/src/dom/coordinates.ts#L174-L179
) logic that could be impacted by this.

These tests also seem to be influenced by how the AUT is being scaled; I can get them to pass/fail by playing with the AUT size in Firefox. I wonder if in 10.0-release the dom updates + AUT scale differences in run mode are different enough to throw this off in CI.

Rather than adding fuzzy assertions, you could try moving the logic to generate the reference `fromElWindow` value into the `.then()`; the `.get()` scrolls the button into view, which I think is what introduces the discrepancy. I don't think doing that impacts the validity of the test at all and should get you a consistent result.

Reply:
@tbiethman something like 3715d3e7dc you think will do the trick?